pages/docs/reference/whatsnew14.md

```diff
@@ -824,6 +820,48 @@ fun main() {
 ```
 </div>
 
+### Functions for string manipulations
+
+The standard library in 1.4 includes a number of improvements in the API for string manipulation:
+
+* `StringBuilder` has new useful extension functions: `set()`, `setRange()`, `deleteAt()`, `deleteRange()`, `appendRange()`
+and others.
+    <div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.4">
+    
+    ```kotlin
+    fun main() {
+    //sampleStart
+        val sb = StringBuilder("Bye Kotlin 1.3.72")
+        sb.deleteRange(0, 3)
+        sb.insertRange(0, "Hello", 0 ,5)
+        sb.set(15, '4')
+        sb.setRange(17, 19, "0")
+        print(sb.toString())
+    //sampleEnd
+    }
+    ```
+    </div>
+    
+* Some existing functions of `StringBuilder` are available in the common library. Among them are `append()`, `insert()`,
+`substring()`, `setLength()`, and more. 
+* New functions `Appendable.appendLine()` and `StringBuilder.appendLine()` are added to the common library. They replace
+JVM-only `appendln()` functions of these classes.
+    <div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.4">
+    
+    ```kotlin
+    fun main() {
+    //sampleStart
+        println(buildString {
+            appendLine("Hello,")
+            appendLine("world")
+        })
+    //sampleEnd
+    }
+    ```
+    </div>
+    
+* `StringBuilder.append()` now accept nullable arguments.
```

Comment:
"nullable **string** argument."
Previously `append()` could also accept nullable argument via `append(value: Any?)` and `append(value: CharSequance?)`.

Changing from `append(value: String)` to `append(value: String?)` is not imperceptible to users. 
Now `sb.append(nullableStringVar)` resolves to `append(value: String?)` instead of `append(value: CharSequance?)`, but behaviour of the call doesn't change.

I suggest not to mention this change.